src/dist/replication/meta_server/meta_service.cpp

```diff
@@ -923,5 +926,15 @@ void meta_service::on_app_partition_split(app_partition_split_rpc rpc)
                      server_state::sStateHash);
 }
 
+void meta_service::on_register_child_on_meta(register_child_rpc rpc)
+{
+    RPC_CHECK_STATUS(rpc.dsn_request(), rpc.response());
+
+    tasking::enqueue(LPC_META_STATE_NORMAL,
+                     tracker(),
+                     [this, rpc]() { _split_svc->register_child_on_meta(std::move(rpc)); },
+                     server_state::sStateHash);
```

Comment:
So multiple `register_child_rpc ` requests must be executed sequentially in a single thread?

Reply:
Yes, threadpool `THREAD_POOL_META_STATE` worker_count is 1. All meta state functions will run in such thread.